tests/test_utils_log_handler.py

```diff
@@ -0,0 +1,29 @@
+from logging import FileHandler
+from logging.handlers import TimedRotatingFileHandler
+from scrapy.utils import log
+import unittest
+
+
+class TestCase(unittest.TestCase):
+
+
+    def test_import_handler(self):
+        """Test function: _import_handler"""
+        name = 'logging.handlers.TimedRotatingFileHandler'
+        self.assertEqual(log._import_hander(name), TimedRotatingFileHandler)
+
+    def test_get_handler(self):
+        """Test function: _get_handler with LOG_FILE"""
+        # First create log file path
+        import os
+        import tempfile
+        file_path = tempfile.mkstemp()[1]
+        settings = {'LOG_FILE': file_path, 'LOG_ENCODIUNG': 'utf-8', 'LOG_LEVEL': 'DEBUG'}
```

Comment:
Noob question, does this mean scrapy will support TimedRotatingFileHandler soon? If true, when is this update coming?  And any pointers for what should I do before this update? ...